packages/skip/src/entry_point.rs

```diff
@@ -26,6 +26,14 @@ pub struct InstantiateMsg {
 #[cw_serde]
 #[allow(clippy::large_enum_variant)]
 pub enum ExecuteMsg {
+    AxelarSwapAndAction {
+        user_swap: Swap,
+        min_coin: Coin,
+        timeout_timestamp: u64,
+        post_swap_action: Action,
+        affiliates: Vec<Affiliate>,
+        recovery_addr: Addr,
```

Comment:
I believe best practice is to accept String and convert it into a check addr after verifying it's validity, but in this case it may not matter much because if we did do a validity check on the address and it came back as non-valid I think the only thing we can do there is fail the tx, which would be what similarly happened if the bank send is sent with an invalid address.

Nothing actionable with this comment, just noting for later in case we did want to do something if this was the case (like keeping the funds on contract, etc.)